lib/ApiResource.php

```diff
@@ -99,8 +99,7 @@ protected static function _validateParams($params = null)
             $message = "You must pass an array as the first argument to Stripe API "
                . "method calls.  (HINT: an example call to create a charge "
                . "would be: \"Stripe\\Charge::create(array('amount' => 100, "
-               . "'currency' => 'usd', 'card' => array('number' => "
-               . "4242424242424242, 'exp_month' => 5, 'exp_year' => 2015)))\")";
+               . "'currency' => 'usd', 'source' => 'tok_visa'))\")";
```

Comment:
Minor, but should we change this to something like `tok_123` to help suggest that this isn't supposed to be a static string of some sort?